aries-site/src/layouts/navigation/SearchResults.js

```diff
@@ -0,0 +1,168 @@
+import { useContext, useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Keyboard,
+  List,
+  ResponsiveContext,
+  Pagination,
+  Text,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+import { SearchInput, SearchResult } from '.';
+
```

Comment:
was this added so that when you searched it would show on first page?

Reply:
Not sure which line you have tagged here, but guessing:
```
const defaultPage = { begin: 0, end: 4 };
```
?

If so, these are required to make the controlled paginated results work correctly.

Reply:
Yes it was those line I was just curious how you fixed that!

Reply:
Since we are using controlled pagination, then we also need to control which subset of results are passed to List's data. This is the mechanism to do those slices of results.